.github/workflows/publish-fig-spec.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Fig Completion Spec
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+  release:
+    types:
+      - published
```

Comment:
git diffing after a publish probably isn't a reliable way to check if something has changed. You'd probably have to execute the command on winglang@V2 and winglang@V1 and diff them.

Also, why not just generate the spec during the build? That way we don't have to execute anything here and you can just use https://docs.npmjs.com/cli/v7/commands/npm-diff to check if anything changed.